report/2.Analysis/EmbeddedPlatforms.tex

```diff
@@ -75,14 +76,15 @@ \subsection{Takeaways}
 		Raspberry Pi Zero       & 32bit ARM1176ZF single core        & 1 GHz   & 512 MB & MicroSD slot
 	\end{tabular}
 \end{table}
+%TODO Fiks den tabel her
 
 The Raspberry Pi system can be quite powerful a project regarding embedded systems, which can also be said about the EV3.
-This can be a disadvantage, as we aren't fully in control of the execution order.
+This can be a disadvantage, as we are not fully in control of the execution order.
+%TODO Vi skal skrive noget om execution order senere, måske i RTS theory afsnittet
+%TODO Giv en reference til RTS afsnittet når det er skrevet
 
 The Arduino and NXT both have less processing power than the Raspberry Pi and EV3, and thus making them more interesting choices, by limiting the hardware capabilities.
-The disadvantage of the Arduino has fewer supported programming languages, and the fact that the GPIO ports make for hardware-related challenges, which is not the purpose of this semester.
+The disadvantage of the Arduino is that it has fewer supported programming languages, and the fact that the GPIO ports make for hardware-related challenges, which is not the purpose of this semester.
 Unlike the Arduino, the NXT offers easy to use hardware ports, and large variety of sensors and motors.
 As the NXT is supplied by the university it makes for the obvious choice, especially considering the points provided in this section.
-
-
-%Figure out if we actually wanna make a choice here, or wait till we have looked at sensors.
+%TODO Figure out if we actually wanna make a choice here, or wait till we have looked at sensors.
```

Comment:
Old todo: Figure out if we actually wanna make a choice here, or wait till we have looked at sensors.

Reply:
det vil vi gerne